include/kimera-vio/utils/ColorUtils.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <opencv2/core/core.hpp>
+
+namespace VIO {
+
+/* Color namespace is used to conveniently produce cv color values. The actual
+   Color values are taken from cv::viz:
+   https://github.com/apc-llc/opencv-2.4.10/blob/master/modules/viz/include/opencv2/viz/types.hpp#L191
+
+   cv::viz is a heavy component to include as a mandatory dependency for Color
+   values only, hence the reason for this inclusion.
+*/
+namespace Color {
+  inline cv::Scalar Apricot() { return cv::Scalar(177, 206, 251); }
+  inline cv::Scalar Black() { return cv::Scalar(0, 0, 0); }
+  inline cv::Scalar Blue() { return cv::Scalar(255, 0, 0); }
+  inline cv::Scalar Brown() { return cv::Scalar(0, 75, 150); }
+  inline cv::Scalar Green() { return cv::Scalar(0, 255, 0); }
+  inline cv::Scalar Pink() { return cv::Scalar(203, 192, 255); }
+  inline cv::Scalar Purple() { return cv::Scalar(128, 0, 128); }
+  inline cv::Scalar Red() { return cv::Scalar(0, 0, 255); }
+  inline cv::Scalar White() { return cv::Scalar(255, 255, 255); }
+
+  inline cv::Vec3b ScalarToVec3b(cv::Scalar value) {
```

Comment:
Based on the [file that you reference at line 198](https://github.com/apc-llc/opencv-2.4.10/blob/master/modules/viz/include/opencv2/viz/types.hpp#L198) it looks like you can just pass the Scalar value as a parameter to Vec3b constructor. Did you already try that and it not work?

Reply:
The intention is to completely remove that OpenCV module as a dependency, so that function would be available in those cases. 

The reason for removing cv::viz it that is has many large dependencies that we don't need in a production environment or low-powered devices.

Reply:
The suggestion wasn't to use the function from the module but to simplify this implementation to be like that one because unless I am mistaken [cv::Scalar_< _Tp >](https://docs.opencv.org/3.4/d1/da0/classcv_1_1Scalar__.html) should have [a public property val](https://docs.opencv.org/3.4/de/de1/classcv_1_1Matx.html#afa9ad3bc5b09ebcdfc6f98f44c15191d) from its ancestor [cv::Matx< _Tp, m, n >](https://docs.opencv.org/3.4/de/de1/classcv_1_1Matx.html) which could have been passed to the cv::Vec3b constructor like they did in the inline function I referenced.

Reply:
Another quick optimisation, we should pass cv::Scalar by reference.

```suggestion
  inline cv::Vec3b ScalarToVec3b(const cv::Scalar& value) {
```

Reply:
@foohyfooh tested this and it didn't work and that was how I remembered that I tried that before as well. The double values needed to be casted to char.